lua/core/builtins/init.lua

```diff
@@ -0,0 +1,25 @@
+local M = {}
+
+local builtins = {
+  require "keymappings",
+  require "core.which-key",
+  require "core.status_colors",
+  require "core.gitsigns",
+  require "core.compe",
+  require "core.dashboard",
+  require "core.dap",
+  require "core.terminal",
+  require "core.telescope",
+  require "core.treesitter",
+  require "core.nvimtree",
+  require "core.rooter",
+  require "core.bufferline",
+}
+
+function M.config(config)
+  for _, builtin in ipairs(builtins) do
+    builtin.config(config)
```

Comment:
It's the teeniest bit cleaner to remove the require statements from above and change this line to ` require(builtin).config(config)`